ITST_DESC : {
    FILE_DIR : "/home/amsl/Pictures/ros_catkin_ws/ref_data_manager",
    FILE_DIR2 : "/sample",
    FILE_DIR3 : "/itst_desc",
    FILE_EXT : ".csv"
}
